pkg/model/config.go

```diff
@@ -120,9 +120,9 @@ type BackupPolicy struct {
 	Storage            *string     `yaml:"storage,omitempty" json:"storage,omitempty"`
 	Namespace          *string     `yaml:"namespace,omitempty" json:"namespace,omitempty"`
 	Parallel           *int32      `yaml:"parallel,omitempty" json:"parallel,omitempty"`
-	SetList            *[]string   `yaml:"set_list,omitempty" json:"set_list,omitempty"`
-	NodeList           *string     `yaml:"node_list,omitempty" json:"node_list,omitempty"`
-	BinList            *string     `yaml:"bin_list,omitempty" json:"bin_list,omitempty"`
+	SetList            []string    `yaml:"set_list,omitempty" json:"set_list,omitempty"`
+	BinList            []string    `yaml:"bin_list,omitempty" json:"bin_list,omitempty"`
+	NodeList           []string    `yaml:"node_list,omitempty" json:"node_list,omitempty"`
```

Comment:
Use this field in the backup configuration.

Reply:
I think, it would be better to have structure for Node